lib/common/results.c

```diff
@@ -22,6 +22,9 @@
 #include <crm/common/mainloop.h>
 #include <crm/common/xml.h>
 
+G_DEFINE_QUARK("pcmk-rc-error-quark", pcmk__rc_error)
+G_DEFINE_QUARK("pcmk-exitc-error-quark", pcmk__exitc_error)
```

Comment:
from the docs: "the quark name will be stringified automatically in the macro, so you shouldn't use double quotes"

Reply:
Fixed.